packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -178,7 +178,7 @@ export abstract class BigValueLayout {
     // The graph field configuration applied to Y values
     const config: FieldConfig<GraphFieldConfig> = {
       custom: {
-        drawStyle: DrawStyle.Line,
+        drawStyle: GraphDrawStyle.Line,
```

Comment:
Can renaming enums be problematic for older plugins?

Reply:
I think this is a revert back to the original name, correct @ryantxu @sdboyer?

Reply:
Not according to this https://github.com/grafana/grafana/blob/41f0542c1ec16ce93b336a9f5cf6eef1aba898d0/packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx#L12

Reply:
I guess the exported name from `@grafana/ui` is still the same?
https://github.com/grafana/grafana/pull/38518/files#diff-a5c51aa022a3253d89b1b504b9eaca6ba8f2601fadd29d70c591c87125321741R7

Reply:
Right existing plugins keep "DrawStyle" import, but I think we want a prefix for that since it is in the global scope.  This PR aims to show how we can support this migration selectively